src/uv-iiif-config.json

```diff
@@ -33,7 +34,9 @@
     },
     "openSeadragonCenterPanel": {
       "options": {
-        "autoHideControls": false
+        "autoHideControls": false,
+        "showHomeControl": true,
+        "showAdjustImageControl": true

```

Comment:
I see that showAdjustImageControl is set to true here, but it is set to false in the OpenSeadragon configuration, and it also appears to default to false. I haven't investigated how all of these settings interact (no hands-on testing yet), but I wonder if we'd be better off making it consistent everywhere. I'd also be inclined to turn this on by default, since it's a useful new feature that shouldn't harm anything. People can turn it off if they see it and don't want it, but if we leave it off by default, they may never know that it's there to turn on.

Reply:
Yeah, I was a bit unsure how all those config.json files work, but setting them to 'true' in them as well seems like a good idea, I'll fix that.